Makefile.gen.kb.mk

```diff
@@ -0,0 +1,108 @@
+
```

Comment:
This is generated by `kubebuilder init`. We will need to figure out how to manage it with `giantswarm/github`, but this should be fine for now.